charts/consul/templates/sync-catalog-deployment.yaml

```diff
@@ -32,6 +32,9 @@ spec:
         {{- if .Values.syncCatalog.extraLabels }}
           {{- toYaml .Values.syncCatalog.extraLabels | nindent 8 }}
         {{- end }}
+        {{- if .Values.global.extraLabels }}
```

Comment:
In this file, we need to add extraLabels to `.metadata.Labels` as well.

Reply:
Ah, good catch! I wrote tests for that bit too but my bats filtering accidentally excluded that one from running...

All tests for affected files pass on my end now, at least